environment

```diff
@@ -29,3 +29,11 @@ for i in "${CFU_ROOT}/third_party/python"/* "${CFU_ROOT}/python/"* "${CFU_ROOT}/
 done
 export PYTHONPATH
 
+
+if [ -d "${CFU_ROOT}/third_party/usr/local/bin" ] ; then
```

Comment:
Equivalent code using bash substring testing (idea via stackoverflow, iirc):

    pathadd() {
        if [ -d "$1" ] && [[ ":$PATH:" != *":$1:"* ]]; then
            export PATH="${1}${PATH:+":$PATH"}"
            echo "Updated PATH: ${PATH}"
        fi
    }
    pathadd "${CFU_ROOT}/third_party/usr/local/bin"

Using a function helps reduce clutter by replacing `${CFU_ROOT}/third_party/usr/local/bin` with a shorter variable name

Reply:
Thanks Alan, I'll make this change.